deployment/chainloop/README.md

```diff
@@ -334,6 +350,21 @@ secretsBackend:
     serviceAccountKey: [KEY]
 ```
 
+### Use Azure KeyVault as credentials backend
```

Comment:
Why don't you follow the convention from previous sections?

I would either say "Use Azure secrets manager" or "Use Azure KeyVault"